### ISSUE TITLE
analyze: empty stats

### ISSUE THREAD

**Issue by bsr203** [mxstbr/react-boilerplate#798]
Hi,

I tried, `analyze` command, but the generated file is empty

```
~/W/G/react-boilerplate ❯❯❯ cat stats.json                                                                                                                                                master
{
  "errors": [],
  "warnings": []
}
```

I tried running it to see the output like,

```
~/W/G/react-boilerplate ❯❯❯ ./node_modules/.bin/webpack --config internals/webpack/webpack.prod.babel.js --profile --json                                                                 master
{
  "errors": [],
  "warnings": []
}
~/W/G/react-boilerplate ❯❯❯ git status                                                                                                                                                    master
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean
~/W/G/react-boilerplate ❯❯❯               
```

results the same. 

I just cloned the repo, so it should be reproducible in the master.

**Comment by bsr203** [comment]
Hi,

It works fine if I comment out `stats` like below in `webpack.base.babel.js`. so we may have to make it configurable.

```
  devtool: options.devtool,
  target: 'web', // Make web variables accessible to webpack, e.g. window
  // stats: false, // Don't show stats in the console
  progress: true,
```

**Comment by mxstbr** [comment]
Woah, that's weird! Why would that make a difference?

**Comment by mxstbr** [comment]
Fixed in #1032 by the amazing @chaintng
